test/edu/ucsb/nceas/MCTestCase.java

```diff
@@ -311,92 +266,92 @@ protected void queryDocWhichHasTitle(String titlePart, String testTitle,
         }
 
     }
-	
-	/*
-	 * Retrus an access block base on params passed and the defaul perm order -
-	 * allow first
-	 */
-	protected String getAccessBlock(String principal, boolean grantAccess, boolean read,
-			boolean write, boolean changePermission, boolean all) {
-		return getAccessBlock(principal, grantAccess, read, write, changePermission, all,
-				ALLOWFIRST);
-	}
-
-	/**
-	 * This function returns an access block based on the params passed
-	 */
-	protected String getAccessBlock(String principal, boolean grantAccess, boolean read,
-			boolean write, boolean changePermission, boolean all, String permOrder) {
-		String accessBlock = "<access "
-				+ "authSystem=\"ldap://ldap.ecoinformatics.org:389/dc=ecoinformatics,dc=org\""
-				+ " order=\"" + permOrder + "\"" + " scope=\"document\"" + ">";
-
-		accessBlock += generateOneAccessRule(principal, grantAccess, read, write,
-				changePermission, all);
-		accessBlock += "</access>";
-
-		return accessBlock;
-
-	}
-
-	/*
-	 * Gets eml access block base on given acccess rules and perm order
-	 */
-	protected String getAccessBlock(Vector<String> accessRules, String permOrder) {
-		String accessBlock = "<access "
-				+ "authSystem=\"ldap://ldap.ecoinformatics.org:389/dc=ecoinformatics,dc=org\""
-				+ " order=\"" + permOrder + "\"" + " scope=\"document\"" + ">";
-		// adding rules
-		if (accessRules != null && !accessRules.isEmpty()) {
-			for (int i = 0; i < accessRules.size(); i++) {
-				String rule = (String) accessRules.elementAt(i);
-				accessBlock += rule;
-
-			}
-		}
-		accessBlock += "</access>";
-		return accessBlock;
-	}
-
-	/*
-	 * Generates a access rule for given parameter. Note this xml portion
-	 * doesn't include <access></access>
-	 */
-	protected String generateOneAccessRule(String principal, boolean grantAccess,
-			boolean read, boolean write, boolean changePermission, boolean all) {
-		String accessBlock = "";
-
-		if (grantAccess) {
-			accessBlock = "<allow>";
-		} else {
-			accessBlock = "<deny>";
-		}
-
-		accessBlock = accessBlock + "<principal>" + principal + "</principal>";
-
-		if (all) {
-			accessBlock += "<permission>all</permission>";
-		} else {
-			if (read) {
-				accessBlock += "<permission>read</permission>";
-			}
-			if (write) {
-				accessBlock += "<permission>write</permission>";
-			}
-			if (changePermission) {
-				accessBlock += "<permission>changePermission</permission>";
-			}
-		}
-
-		if (grantAccess) {
-			accessBlock += "</allow>";
-		} else {
-			accessBlock += "</deny>";
-		}
-		return accessBlock;
-
-	}
-	
+
+    /*
+     * Retrus an access block base on params passed and the defaul perm order -
```

Comment:
spelling: Returns

Reply:
MCTestCase addressed in commit 186c5d14db0f94725f6a410d4028ca9a37925a39